.gitignore

```diff
@@ -10,3 +10,4 @@ res/version.rc
 res/version.txt
 res/julius.icns
 /CMakeSettings.json
+out/
```

Comment:
Always end the file with a newline (see: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)